source/renderer/app/stores/WalletsStore.js

```diff
@@ -266,6 +272,8 @@ export default class WalletsStore extends Store {
     walletsActions.sendMoney.listen(this._sendMoney);
     walletsActions.importWalletFromFile.listen(this._importWalletFromFile);
     walletsActions.chooseWalletExportType.listen(this._chooseWalletExportType);
+    walletsActions.getWalletPublicKey.listen(this._getWalletPublicKey);
```

Comment:
@aleksandardjordjeviciohk please remove this.

Reply:
Fixed